daos-cortx/docs/mpifileutils_setup.md

```diff
@@ -0,0 +1,430 @@
+
+    Terminal logs -
+        
+         /bin/mkdir -p '/root/mpifileutils_src/install/bin'
+          /bin/sh ./libtool   --mode=install /bin/install -c bsdtar bsdcpio bsdcat '/root/mpifileutils_src/install/bin'
+        libtool: install: /bin/install -c bsdtar /root/mpifileutils_src/install/bin/bsdtar
+        libtool: install: /bin/install -c bsdcpio /root/mpifileutils_src/install/bin/bsdcpio
+        libtool: install: /bin/install -c bsdcat /root/mpifileutils_src/install/bin/bsdcat
+         /bin/mkdir -p '/root/mpifileutils_src/install/include'
+         /bin/install -c -m 644 libarchive/archive.h libarchive/archive_entry.h '/root/mpifileutils_src/install/include'
+         /bin/mkdir -p '/root/mpifileutils_src/install/share/man/man1'
+         /bin/install -c -m 644 tar/bsdtar.1 cpio/bsdcpio.1 cat/bsdcat.1 '/root/mpifileutils_src/install/share/man/man1'
+         /bin/mkdir -p '/root/mpifileutils_src/install/share/man/man3'
+         /bin/install -c -m 644 libarchive/archive_entry.3 libarchive/archive_entry_acl.3 libarchive/archive_entry_linkify.3 libarchive/archive_entry_misc.3 libarchive/archive_entry_paths.3 libarchive/archive_entry_perms.3 libarchive/archive_entry_stat.3 libarchive/archive_entry_time.3 libarchive/archive_read.3 libarchive/archive_read_add_passphrase.3 libarchive/archive_read_data.3 libarchive/archive_read_disk.3 libarchive/archive_read_extract.3 libarchive/archive_read_filter.3 libarchive/archive_read_format.3 libarchive/archive_read_free.3 libarchive/archive_read_header.3 libarchive/archive_read_new.3 libarchive/archive_read_open.3 libarchive/archive_read_set_options.3 libarchive/archive_util.3 libarchive/archive_write.3 libarchive/archive_write_blocksize.3 libarchive/archive_write_data.3 libarchive/archive_write_disk.3 libarchive/archive_write_filter.3 libarchive/archive_write_finish_entry.3 libarchive/archive_write_format.3 libarchive/archive_write_free.3 libarchive/archive_write_header.3 libarchive/archive_write_new.3 libarchive/archive_write_open.3 libarchive/archive_write_set_options.3 libarchive/archive_write_set_passphrase.3 libarchive/libarchive.3 libarchive/libarchive_changes.3 libarchive/libarchive_internals.3 '/root/mpifileutils_src/install/share/man/man3'
+         /bin/mkdir -p '/root/mpifileutils_src/install/share/man/man5'
+         /bin/install -c -m 644 libarchive/cpio.5 libarchive/libarchive-formats.5 libarchive/mtree.5 libarchive/tar.5 '/root/mpifileutils_src/install/share/man/man5'
+         /bin/mkdir -p '/root/mpifileutils_src/install/lib/pkgconfig'
+         /bin/install -c -m 644 build/pkgconfig/libarchive.pc '/root/mpifileutils_src/install/lib/pkgconfig'
+        make[2]: Leaving directory `/root/mpifileutils_src/deps/libarchive-3.5.1'
+        make[1]: Leaving directory `/root/mpifileutils_src/deps/libarchive-3.5.1'
+        [root@ssc-vm-2051 mpifileutils_src]#
+
+
+# Build mpifileutils source
+
+* Make sure daos setup and building is already done using these steps[setup_daos.md].
```

Comment:
setup_daos.md in current repo or put actual link

Reply:
done